cpg-neo4j/src/main/java/de/fraunhofer/aisec/cpg_vis_neo4j/Application.kt

```diff
@@ -61,6 +66,14 @@ private const val DEFAULT_USER_NAME = "neo4j"
 private const val DEFAULT_PASSWORD = "password"
 private const val DEFAULT_SAVE_DEPTH = -1
 
+data class CompilationDbStructure(
```

Comment:
I think this should probably be move to cpg-core and named something like `CompliationDatabaseEntry` or rather something related to "command objects" as they are called in the original spec.

Reply:
I can see that, are we also using the data class in cpg-core code? if not we may consider doing so and move it there

Reply:
Renamed it to CompliationDatabaseEntry and moved it to `cpg-core/src/main/java/de/fraunhofer/aisec/cpg/frontends/cpp/CompilationDB.kt`.